cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -471,9 +471,10 @@ def test_custom_button_on_vm_resource_detail(context):
             2. In normal UI - OPS, Provision test service using any infra provider
             (nvc55 recommended)
             3. In SSUI, Check custom button on VM resource details page
+            4. Check button execution
```

Comment:
This should be an expectedResult, along with step 3 as well from the looks of it.

Is the validate here just that the button is visible? Should it be clickable? What should the custom button do? Step 1 is not clear here, what the button should be or do.

Please add some more detail here - "check button execution" is very generic, and I think there's an assumption here that the button has been clicked.   Neither step 2 nor step 3 actually instruct the tester to click the button - just to "check" the custom button, which I assume is an expectedResult that the button is visible.

Reply:
@mshriver  I tried to add explanation as per my understanding. Have a glance.